Adding missing action data to the Monster Manual

NeilMartin opened the pull request:
I noticed there were missing actions for the Yuan-Ti Abomination. I scanned for missing actions in the other Monster Manual entries and found that the Bone Devil Polearm and Drider Spellcaster were also missing their actions.
I don't understand what the first number represents in the <action><attack> data, so I simply duplicated the value from similar attacks.
Also, the Bone Devil had duplicate actions, so I removed them.

NeilMartin committed:
Adding missing action data to Monster Manual

NeilMartin commented:
github stripped out my xml "<". I'll try again with "["

...I don't understand what the first number represents in the [actions] [attack] data...

NeilMartin committed:
Added value of mundane items

NeilMartin committed:
Missed some values

ceryliae commented:
Are you referring to the 8 in the following example?

<attack>Claw|8|1d8+4</attack>

NeilMartin commented:
Yes, that's the number I couldn't figure out.

ceryliae commented:
It's the bonus to attack rolls
On Wed, Jun 1, 2016 at 5:09 PM NeilMartin notifications@github.com wrote:

> Yes, that's the number I couldn't figure out.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> https://github.com/ceryliae/DnDAppFiles/pull/207#issuecomment-223161553,
> or mute the thread
> https://github.com/notifications/unsubscribe/ABzda4_IQieAWg0vshnsJ-OUtpEQ0bv8ks5qHh8tgaJpZM4IjdCJ
> .

NeilMartin commented:
Ah right, so d20+8 vs AC for Claw to hit. That makes sense.
I'll have to check the values I've added, they are likely incorrect.

ceryliae commented:
Re resubmit once you've checked please
On Wed, Jun 1, 2016 at 5:14 PM NeilMartin notifications@github.com wrote:

> Ah right, so d20+8 vs AC for Claw to hit. That makes sense.
> I'll have to check the values I've added, they are likely incorrect.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> https://github.com/ceryliae/DnDAppFiles/pull/207#issuecomment-223162378,
> or mute the thread
> https://github.com/notifications/unsubscribe/ABzda96p5xXudZ6b-usXTBPDEuKkMgC_ks5qHiB3gaJpZM4IjdCJ
> .

NeilMartin committed:
Fixed some discrepencies between the +X to hit descriptions and the attack data

NeilMartin commented:
I've fixed the discrepancies I added and I found some more that were already in there. I also found some missing attack data blocks and added those too.
